### PR TITLE
Russian translation updates

### PR DESCRIPTION
**Pull request by bodrovis**
Subject.

### CODE REVIEW

File: config/locales/ru.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ru:
   forem:
     general:
+      deleted: [deleted]
```
Comment (yas375) [inline config/locales/ru.yml]: Should it be `[удалено]`?

**Comment by bodrovis** [comment]
@yas375 Oh, my bad. Of course it should. Will fix in a moment.

**Comment by bodrovis** [comment]
Could you merge this plz? :)

**Comment by radar** [comment]
Added to rails4 and rails3.

**Pull request closed without merging**